### PR TITLE
removed views engine and removed pug dependency as no longer needed. …

### PR DESCRIPTION
**Pull request by Vourpal**
…Added a custom 404 page that will be called whenever an incorrect route is used

## Describe your changes

_Provide description_

Removed Views engine and Pug dependency from package.json as it is no longer used.  A custom 404 page was created for user experience

## Task URL (Asana, Jira, etc...)

_Provide url_

## What was done in this pull request

- [ ] Removed Views engine and Pug
- [ ] Added custom 404 error page

## Demo / Screenshot / Video
image

_Provide demo if possible_

### CODE REVIEW

File: public/404.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/stylesheets/404Style.css" />
+    <link rel="stylesheet" href="/stylesheets/fonts.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1 id="Uh-Oh">Uh Oh!</h1>
```
Comment (arielplas) [inline public/404.html:16]: This should be `h2`

File: public/404.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    <div id="cat-wrapper">
+      <img
+        src="https://i.pinimg.com/736x/c6/80/ea/c680eafa53543372915f778e67a78b7c--cat-posters-hang-in-there.jpg"
+        alt="hanging-cat"
+        id="hanging-cat-img"
+      />
+    </div>
+
+    <h2 id="error-404">404 Error!</h2>
```
Comment (arielplas) [inline public/404.html:25]: this should be `h1`

File: public/404.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/stylesheets/404Style.css" />
+    <link rel="stylesheet" href="/stylesheets/fonts.css" />
+    <title>Document</title>
```
Comment (arielplas) [inline public/404.html]: This should be "Dailycat" and the `favicon` is missing.

**Review by imariel2d — CHANGES_REQUESTED** [review]
Please review requested changes, also please add the navbar :)

File: public/stylesheets/404Style.css [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+#navbar {
+  padding: 1rem;
+  background-color: #f0dbdb;
+}
+
+#page-title {
+  margin: 0;
+  color: #dba39a;
+  font-size: 1.5rem;
+}
```
Comment (arielplas) [inline public/stylesheets/404Style.css]: there's no need to re-do these styles, just add the `style.css` into the html file

File: public/404.html [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="icon" type="image/x-icon" href="/images/dailycat-logo.svg" />
+    <link rel="stylesheet" href="/stylesheets/404Style.css" />
+    <link rel="stylesheet" href="/stylesheets/fonts.css" />
+    <title>Dailycat</title>
+  </head>
+  <body>
+    <nav id="navbar">
+      <h1 id="page-title">Dailycat</h1>
```
Comment (arielplas) [inline public/404.html]: Oh yeah, this was the correct `h1`. Nice.

File: public/404.html [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+    <h2 id="Uh-Oh">Uh Oh!</h2>
+    <div id="cat-wrapper">
+      <img
+        src="https://i.pinimg.com/736x/c6/80/ea/c680eafa53543372915f778e67a78b7c--cat-posters-hang-in-there.jpg"
+        alt="hanging-cat"
+        id="hanging-cat-img"
+      />
+    </div>
+
+    <h1 id="error-404">404 Error!</h1>
```
Comment (arielplas) [inline public/404.html]: Let's change it back to `h2`.

**Review by imariel2d — CHANGES_REQUESTED** [review]
Please review the comments, also let's add a "Go Home", to redirect to `/` in case they fall into this page.

File: public/404.html (hunk 5 above)
Comment (Vourpal) [reply public/404.html]: ur cringe

**Comment by Vourpal** [comment]
wdym review the comments, have i not been responding to all of them o-o?

**Review by imariel2d — APPROVED** [review]

**Pull request merged**